includes/admin.php

```diff
@@ -5,134 +5,207 @@
  * @package SuperSaaS
  */
 
-defined( 'ABSPATH' ) or die( 'No script kiddies please!' );
+defined('ABSPATH') or die('No script kiddies please!');
 
 /**
  * Adds the SuperSaaS settings page.
  */
-function supersaas_add_admin_menu() {
-	add_options_page( __( 'SuperSaaS Settings', 'supersaas' ), 'SuperSaaS', 'manage_options', 'supersaas-settings', 'supersaas_options' );
+function supersaas_add_admin_menu()
+{
+  add_options_page(__('SuperSaaS Settings', 'supersaas'), 'SuperSaaS', 'manage_options', 'supersaas-settings', 'supersaas_options');
 }
 
 /**
  * Registers the SuperSaaS settings.
  */
-function supersaas_register_settings() {
-	register_setting( 'supersaas-settings', 'ss_account_name' );
-	register_setting( 'supersaas-settings', 'ss_password' ); // NOTE: this is an API KEY, not a user password; the "ss_password" key is used for backwards compatibility
-	register_setting( 'supersaas-settings', 'ss_schedule' );
-	register_setting( 'supersaas-settings', 'ss_button_label' );
-	register_setting( 'supersaas-settings', 'ss_button_image' );
-	register_setting( 'supersaas-settings', 'ss_domain', 'domain_from_url' );
+function supersaas_register_settings()
+{
+  register_setting('supersaas-settings', 'ss_account_name');
+  register_setting('supersaas-settings', 'ss_display_choice'); // new
+  register_setting('supersaas-settings', 'ss_autologin_enabled'); // new
+  register_setting('supersaas-settings', 'ss_password'); // NOTE: this is an API KEY, not a user password; the "ss_password" key is used for backwards compatibility
```

Comment:
If customers update their plugin from v1 to v2 will it break? If it works the great, otherwise we may as well rename this to ss_api_key.